website/content/en/docs/usage/metrics.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Metrics"

```

Comment:
Can we change this to "Operator Metrics" to be clear (as this is clubbed under "Usage" where application related processes are documented)?

Reply:
Done.. the document is still not up to mark, will work on that separately later.